src/ontology/edits/oeo-physical.omn

```diff
@@ -6138,20 +6176,27 @@ Class: OEO_00030002
 
 In real world, fossils are from biogenic origin some thousands or millions of years ago. However, this is irrelevant in the energy modelling domain.",
         <http://purl.obolibrary.org/obo/IAO_0000233> "issue: https://github.com/OpenEnergyPlatform/ontology/issues/108
-pull request: https://github.com/OpenEnergyPlatform/ontology/pull/130",
+pull request: https://github.com/OpenEnergyPlatform/ontology/pull/130
+
+Update definition:
```

Comment:
Did you really update the definition?
I think you added the 'DisjointWith', right?
I just saw, that in the current definition of `fossil` it is missing that it is a subclass of `geogenic`.